warehouse/templates/includes/flash-messages.html

```diff
@@ -16,7 +16,7 @@
 <div class="notification-bar notification-bar--warning" data-controller="notification" data-target="notification.notification">
   <div class="site-container">
```

Comment:
1. can we please remove the 'site container'?   I understand that you did this to position the close button near the middle of the page, but I fear that it will make some of our longer messages wrap over two lines.

Reply:
Sure thing. I'm going to assume having the close button at the top right is what we're expecting now.